component/core/src/main/java/org/exoplatform/social/core/jpa/storage/dao/jpa/MetadataItemDAO.java

```diff
@@ -55,6 +55,8 @@ public class MetadataItemDAO extends GenericDAOJPAImpl<MetadataItemEntity, Long>
   
   private static final String SPACE_ID             = "spaceId";
 
+  private static final String UNREAD_METADATA_TYPE = METADATA_TYPE_PARAM;
```

Comment:
why repeating the constant, you can reuse it instead IMO.

Reply:
Done!